documentation/docs/developers/building-and-deploying-to-ios.md

```diff
@@ -0,0 +1,59 @@
+# Building and deploying to iOS
+
+Building and deploying an app to iOS must be performed by a developer on a Mac computer. This guide does not include the process for managing the app within Apple's App Store Connect console, which is documented elsewhere.
+
+## Prerequisites
+
+### Apple Developer Account
+
+In order to create iOS applications, you will need to enrol in the [Apple Developer Program](https://developer.apple.com/programs/), which can be done as an individual or organisation.
+
+### XCode
+
+In order to run applications locally on an iOS device or simulator (Apple's term for an iOS device emulator), and submit them to the App Store, you will need XCode. [XCode](https://developer.apple.com/xcode/) is available from the Mac App Store.
+
+You will also need XCode CLI tools. These may have been installed with XCode, run `xcode-select -v` to check. If not already installed, run `xcode-select --install`. The CLI tools may need to be configured using the command `sudo xcode-select -s /Applications/Xcode.app/Contents/Developer`.
+
+### CocoaPods
+
+[CocoaPods](https://cocoapods.org/) is a dependency manager for iOS projects, and is required to build iOS apps via Capacitor. Installing via [Homebrew](https://brew.sh/) seems to be the best method, via the command `brew install cocoapods`.
+
+### Capacitor iOS setup
+
+The project is already configured to build to iOS using Capacitor.
```

Comment:
nit(non-blocking): may be useful to link to capacitor ios docs here as they have a bunch of extra screenshots:
https://capacitorjs.com/docs/ios#running-your-app